### ISSUE TITLE
Support ranged enumeration values

### ISSUE THREAD

**Issue by 5nord** [nokia/ntt#638]
Support enumerated labels which represent multiple values. 

Example from the [standard](https://www.etsi.org/deliver/etsi_es/201800_201899/20187301/04.14.01_60/es_20187301v041401p.pdf#page=64):
```
type enumerated MyThirdEnumType {
    Blue(0),
    Yellow(1),
    Green(3),
    Other(2, 4..255)
}
```

_Originally posted by @5nord in https://github.com/nokia/ntt/pull/637#discussion_r984249376_

**Comment by UrbanskiDawid** [comment]
this is already supported:

```
		{`type enumerated MyThirdEnumType {Blue(0),Yellow(1),Green(3),Other(2, 4..255)}; MyThirdEnumType`,
			NewEnumTypeWithIds(
				"MyThirdEnumType",
				"Blue", []runtime.EnumRange{{First: 0, Last: 0}},
				"Yellow", []runtime.EnumRange{{First: 1, Last: 1}},
				"Green", []runtime.EnumRange{{First: 3, Last: 3}},
				"Other", []runtime.EnumRange{{First: 2, Last: 2}, {First: 4, Last: 255}},
			),
		},
```

passes

**Comment by 5nord** [comment]
Implemented in #637
